src/nrtk_explorer/app/transforms.py

```diff
@@ -161,23 +164,24 @@ def delete_meta_state(old_ids, new_ids):
 
         self._on_transform_fn = None
 
-        self._transforms: Dict[str, trans.ImageTransform] = {
-            "blur": trans.GaussianBlurTransform(),
-            "invert": trans.InvertTransform(),
-            "downsample": trans.DownSampleTransform(),
-            "identity": trans.IdentityTransform(),
+        self._transform_classes: Dict[str, type[trans.ImageTransform]] = {
+            "blur": trans.GaussianBlurTransform,
+            "invert": trans.InvertTransform,
+            "downsample": trans.DownSampleTransform,
+            "identity": trans.IdentityTransform,
         }
 
         if nrtk_trans.nrtk_transforms_available():
-            self._transforms["nrtk_pybsm"] = nrtk_trans.NrtkPybsmTransform()
+            self._transform_classes["nrtk_pybsm"] = nrtk_trans.NrtkPybsmTransform
 
         # Add transform from YAML definition
-        self._transforms.update(nrtk_yaml.generate_transforms())
+        self._transform_classes.update(nrtk_yaml.generate_transforms())
 
-        self._parameters_app._transforms = self._transforms
+        self._parameters_app._transform_classes = self._transform_classes
 
-        self.state.transforms = [k for k in self._transforms.keys()]
-        self.state.current_transform = self.state.transforms[0]
+        # Initialize the transforms pipeline to the identity
+        self._parameters_app._default_transform = "identity"
```

Comment:
What do you think about defaulting to a transform that changes the image?

Reply:
which transform should we choose though? I picked identity cause it was a safe/neutral/boring option :)

Reply:
Blur perhaps?  Something fast and has interesting effect but not bonkers like invert.

Reply:
Also FYI, `blur` would be picked as the default transform when we add a new transform to the chain